common/src/main/java/tachyon/util/UnderFileSystemUtils.java

```diff
@@ -73,4 +72,8 @@ public static void touch(final String path, TachyonConf tachyonConf) throws IOEx
     OutputStream os = ufs.create(path);
     os.close();
   }
+
+  private UnderFileSystemUtils() {
```

Comment:
``private UnderFileSystemUtils() {} // prevent instantiation`` is fewer lines of code and more informative